packages/hurl/src/http/client.rs

```diff
@@ -435,6 +435,9 @@ impl Client {
         if let Some(s) = options.no_proxy.clone() {
             self.handle.noproxy(s.as_str())?;
         }
+        if let Some(unix_socket) = options.unix_socket.clone() {
```

Comment:
A "better" code (without the `clone()`) could be:

```rust
if let Some(unix_socket) = &options.unix_socket {
    self.handle.unix_socket(unix_socket)?;
```

This should make clippy happy. I will correct (in another PR) the sibling code for `options.no_proxy.clone() {` taht could be improve also

Reply:
Nice!